lib/node/index.ts

```diff
@@ -67,7 +67,7 @@ function plugin(options: Options = {}): Plugin {
     );
   }
 
-  const prefix = `.${path.sep}${path.relative(process.cwd(), dir)}`;
+  const prefix = `/${path.sep}${path.relative(process.cwd(), dir)}`;
```

Comment:
`path.sep` is already `/` or `\` depending on the OS. When I try it this creates a string like `//app/routes` and imports will fail.

As far as I can see, this suggestion works on Windows and macOS:
```suggestion
  const prefix = `.${path.sep}${path.relative(process.cwd(), dir)}`.replace(/\\/g, '/');
```

Does that work for you too?

Reply:
I think that my line is wrong. Sorry. I think it should be 
```
const prefix = `./${path.relative(process.cwd(), dir)}`;
```
The problem is the `${path.sep}` not the `.`. 

IMHO your proposal should produce the same result:
```
const prefix = `./${path.relative(process.cwd(), dir)}`.replace(/\\/g, '/');
```

I am wondering if only the replace `remix.ts` line 79 `file: route.file.replace(/\\/g, '/')` could be enough.

Let me check it with my college and go back to you. I am not using windows.

Reply:
What you suggest would work in case `appDirectory` is not set (so it uses `app` as fallback) or it's another single-word directory. But if the user configures it to be `my\\nested\\app`, then we'd have `./my\\nested\\app` and it would break again.
I think most people would use `/` in a node context, even on Windows but if someone uses `path.resolve` or something similar, it will have `\` separators instead of `/` and it will crash.

So maybe it's best to just put `.replace(...)` after the path. The entire string is then OS specific and right at the end, we normalize it.

Reply:
Good point. I think that the mixup is that we are using an OS filesystem separator but at the end, it generates code for Node which is always using `/` for any OS.

Would you mind making a commit with the option that this kit is better and I test it in my college's environment where we found the issue?

Reply:
I have pushed a commit that only changes the path in the utils file once.
It also uses `.split(path.sep).join(path.posix.sep)` instead of `.replace(/\\/g, '/')` as it's a bit more explicit about what it's doing.

Reply:
Looks good and more clear :) It works fine on my macOS environment. Let me test it on the Windows environment and go back to you.

Reply:
The testing was easy. It worked like a charm.
This is the JS generated on Windows:
<img width="1200" alt="Screenshot 2022-10-31 at 09 03 28" src="https://user-images.githubusercontent.com/24297787/198960953-86392eaf-fe47-4e37-b759-1a60c08ca313.png">

Reply:
Great. Thanks for the PRs!